lms/static/js/bookmarks/models/bookmark.js

```diff
@@ -13,7 +13,24 @@
             },
 
             blockUrl: function () {
-                return '/courses/' + this.get('course_id') + '/jump_to/' + this.get('usage_id');
+                return '/courses/' + encodeURIComponent(this.get('course_id')) + '/jump_to/' +
+                    encodeURIComponent(this.get('usage_id'));
+            },
+
+            breadcrumbTrail: function () {
+                var html = HtmlUtils.HTML(
+                    ' <i class="icon fa fa-caret-right" aria-hidden="true"></i><span class="sr">-</span> '
+                );
+                var display_names = _.pluck(this.get('path'), 'display_name');
+                display_names.push(this.get('display_name'));
+
+                return HtmlUtils.joinHtml(
+                    HtmlUtils.ensureHtml(display_names[0]),
```

Comment:
1. `HtmlUtils.ensureHtml()` is not necessary because `joinHtml` will escape anything that is not an Html Snippet.
2. Something seems odd about coding this with `[0]`, `[1]`, and `[2]` referenced directly.  Some possible alternative (untested) code:
```
var breadcrumbs = display_names[0]
for (index = 1; index < display_names.length; index++) {
    breadcrumbs = HtmlUtils.joinHtml(
        breadcrumbs,
        html
        display_names[index]
    );
}
``` 
You can be the judge regarding the second point.